src/components/shared/form/Input.tsx

```diff
@@ -19,6 +19,13 @@ const Input = ({
   errorMessage,
   className,
 }: Props) => {
+  const disablePastDate = () => {
+    const today = new Date();
+    const dd = String(today.getDate() + 1).padStart(2, "0");
+    const mm = String(today.getMonth() + 1).padStart(2, "0"); //January is 0!
+    const yyyy = today.getFullYear();
+    return yyyy + "-" + mm + "-" + dd;
+  };
```

Comment:
You can use momentjs for this https://momentjs.com/docs/#/query/is-before/

Reply:
Thanks for the reference, code updated